client/post-editor/editor-title/index.jsx

```diff
@@ -74,7 +76,7 @@ const EditorTitle = React.createClass( {
 			title: event.target.value
 		} );
 
-		this.props.setTitle( event.target.value );
+		this.props.setTitle( this.props.site.ID, this.props.post.ID, event.target.value );
```

Comment:
I usually don't trust `defaultProps`, often because if we were explicitly pass `null`, it does not replace with the default value. I assume this has been tested such that this won't occur?